api/envoy/data/orca/v1/orca_load_report.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package envoy.data.orca.v1;
+
+option java_outer_classname = "OrcaLoadReportProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.envoy.data.orca.v1";
+option go_package = "v1";
+
+import "validate/validate.proto";
+
+message OrcaLoadReport {
+  // CPU utilization expressed as a fraction of available CPU resources. This
+  // should be derived from a sample or measurement taken during the request.
+  double cpu_utilization = 1 [(validate.rules).double.gte = 0, (validate.rules).double.lte = 1];
+
+  // Memory utilization expressed as a fraction of available memory
+  // resources. This should be derived from a sample or measurement taken
+  // during the request.
+  double mem_utilization = 2 [(validate.rules).double.gte = 0, (validate.rules).double.lte = 1];
+
+  // Application specific requests costs. Values may be absolute costs (e.g.
```

Comment:
Do you want to switch to the updates working in https://docs.google.com/document/d/1NSnK3346BkBo1JUU3I9I5NYYnaJZQPt8_Z_XCBCI3uA/edit#?

Also, can renumber this proto to 3.